specs/enhance_gluster_cluster_sync_logic.adoc

```diff
@@ -0,0 +1,307 @@
+= Enhance gluster cluster synchronization logic as per recommendations
+
+The intent of this change is to accommodate additional details available as part
+of `get-state` output while syncing the gluster cluster details and store them
+in central store.
+
+Also instead of using `gstatus` for cluster health and volume utilization
```

Comment:
@anmolbabu , FYI Please make sure collectd plugins doesn't have dependency to gstatus going forward